ocaml/xapi/quicktest_cbt.ml

```diff
@@ -70,6 +73,40 @@ let enable_disable_cbt_test ~session_id ~vDI =
   | Test_failed msg -> failed enable_cbt_test msg
   | e -> report_failure e enable_cbt_test
 
+(* Test data_destroy and snapshot update the necessary fields *)
```

Comment:
We discussed that we could remove the enable/disable test, because this test includes that one.